specification/esql/async_query/AsyncQueryRequest.ts

```diff
@@ -58,7 +58,11 @@ export interface Request extends RequestBase {
      */
     drop_null_columns?: boolean
     /**
-     * A short version of the Accept header, for example `json` or `yaml`.
+     * A short version of the Accept header, e.g. json, yaml.
+     *
+     * `csv`, `tsv`, and `txt` formats will return results in a tabular format, excluding other metadata fields from the response.
+     *
+     * For async requests, nothing will be returned if the async query doesn't finish within the timeout.
```

Comment:
Maybe "if the request doesn't finish you find the endpoint from the ADFAFADSFDAS header."

Reply:
Added a line about using the headers instead, and some docs on the response object about the headers